### PR TITLE
bugfix: disable defaultSorting when sorting = false

### PR DESCRIPTION
**Pull request by sshanzel**
## Related Issue
#1223 

## Description
Config for validating when applying `defaultSort` is not checking the options for `sorting: false`. Also when applying it directly at the `column`. As per the documentation, this should not be the case.

## Impacted Areas in Application
Sorting of columns.

## Additional Notes
Just let me know if you have any concerns so I can immediately coordinate.

### CODE REVIEW

File: src/material-table.js [hunk 1]
```diff
@@ -55,8 +55,8 @@ export default class MaterialTable extends React.Component {
   setDataManagerFields(props, isInit) {
     let defaultSortColumnIndex = -1;
     let defaultSortDirection = '';
-    if (props) {
-      defaultSortColumnIndex = props.columns.findIndex(a => a.defaultSort);
+    if (props.options.sorting !== false) {
```
Comment (mbrn) [inline src/material-table.js]: You may check for props is not undefined. // (props && props.options.sorting !== false)

Comment (sshanzel) [reply src/material-table.js]: Though i've tested passing nothing on the props and it prompted nothing, yes i'd apply now safe checking. thanks a lot for the review!

**Comment by sshanzel** [comment]
Good day! I've now applied safe checking for `props` when applying `defaultSort`. Thanks a lot for the review!

**Comment by sshanzel** [comment]
You're welcome!

**Pull request merged**